proceeding/tables/catalog_table.tex

```diff
@@ -0,0 +1,21 @@
+% Table
+\begin{table}[h]
+
+\caption{Catalog information.}
+\label{tab:a}
+\tabcolsep7pt\begin{tabular}{ || c|ccc ||}
```

Comment:
I would drop the vertical lines and use alignment `lrll`, i.e. left for text and right for numbers.